spec/abstractops.html

```diff
@@ -1624,8 +1627,9 @@ <h1>
       <dd>It converts _argument_ to an integer representing its Number value (replacing *NaN* with 0), or throws a *RangeError* when that Number value is not a finite integral Number.</dd>
     </dl>
     <emu-alg>
+      1. If _argument_ is *undefined*, return 0.
       1. Let _number_ be ? ToNumber(_argument_).
-      1. If _number_ is *NaN*, *+0*<sub>𝔽</sub>, or *-0*<sub>𝔽</sub>, return 0.
+      1. If _number_ is *-0*<sub>𝔽</sub>, return 0.
```

Comment:
I think we can remove this line altogether instead of keeping the check for negative zero. ℝ(-0<sub>𝔽</sub>) and ℝ(+0<sub>𝔽</sub>) both return 0.